spec/features/tools/delete_tool_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
```

Comment:
don't need this we're actually trying to remove from any files we see this in

Reply:
Will remove. Was copied in from the similar delete_service_spec.rb. Do we have a ticket to remove it from all the other places it is already in?